demosplan/DemosPlanCoreBundle/Command/Db/MigrateCommand.php

```diff
@@ -62,15 +61,14 @@ public function execute(InputInterface $input, OutputInterface $output): int
 
         // ensure that rollup migrations are applied before core migrations are performed
         if (file_exists(DemosPlanPath::getProjectPath($lastRollupMigration))) {
-            $commands[] = $migrationsSyncCommand . $migrationsConfigurationPath . " {$db} --env={$env}";
-            $commands[] = $migrationsCommand . $migrationsConfigurationPath .
-                    'Application\Migrations\Version20220914133419' . " {$db} --env={$env}";
-
+            $commands[] = $migrationsSyncCommand.$migrationsConfigurationPath." {$db} --env={$env}";
+            $commands[] = $migrationsCommand.$migrationsConfigurationPath.
+                    'Application\Migrations\Version20220914133419'." {$db} --env={$env}";
```

Comment:
I think this ok, but personally I find concatination easier to read with blank spaces